furms-integration-tests/src/test/java/io/imunity/furms/integration/tests/security/resource_usage/ResourceUsageServiceSecurityTest.java

```diff
@@ -50,6 +50,38 @@ void shouldPassForSecurityRulesForMethodsInResourceUsageService() {
 						projectAdmin(otherCommunity, otherProject),
 						projectUser(community, project),
 						projectUser(otherCommunity, otherProject))
+			.andForMethods(
+				() -> service.findAllUserUsagesHistory(project, projectAllocation),
+				() -> service.findAllResourceUsageHistory(project, projectAllocation))
+				.accessFor(
+					basicUser(),
+					fenixAdmin(),
+					siteAdmin(site),
+					siteAdmin(otherSite),
+					siteSupport(site),
+					siteSupport(otherSite),
+					communityAdmin(community),
+					communityAdmin(otherCommunity),
```

Comment:
@bihu1 that looks pretty strange (and couple of other lines here. 
Can you ensure it is correct? other community admin should not have access to resource usage of other community project (etc)

Reply:
@golbi This situation is similar like that I've describe above. Those method has PROJECT_LIMITED_READ capability, so all admins can see it, but the main recipients are project user and project admin and community admin.